test/functional/apps/context/index.js

```diff
@@ -21,12 +21,14 @@ export default function ({ getService, getPageObjects, loadTestFile }) {
   const remote = getService('remote');
   const esArchiver = getService('esArchiver');
   const PageObjects = getPageObjects(['common']);
+  const kibanaServer = getService('kibanaServer');
 
   describe('context app', function () {
     before(async function () {
       await remote.setWindowSize(1200, 800);
       await esArchiver.loadIfNeeded('logstash_functional');
       await esArchiver.load('visualize');
+      await kibanaServer.uiSettings.replace({ 'dateFormat:tz': 'UTC', 'defaultIndex': 'logstash-*' });
```

Comment:
I think you want to add the animation setting to off here.  It's possible that would fix it without the sleeps.  Though I know testing that theory will take forever....

Reply:
Oh also, I'm pretty sure `defaultIndex`: `logstash-*` does nothing because we need to use an id. this is used prolifically incorrectly.

Reply:
@Stacey-Gammon This test loads the `visualize` esArchiver data set which contains a logstash-* index pattern with that same id.  But it doesn't contain a config doc.  

```
{
  "type": "doc",
  "value": {
    "index": ".kibana",
    "type": "doc",
    "id": "index-pattern:logstash-*",
    "source": {
      "type": "index-pattern",
      "index-pattern": {
        "title": "logstash-*",
```

The way this has been working prior to this PR is that code in the navigateTo method in common_page checks if where we're navigating to and if it requires a default index pattern, and there isn't one, it sets it and logs some warnings.  (but now I see should also add 
`'telemetry:optIn': false` for cases where we run functional tests against a kibana with x-pack features running).

```
          return kibanaServer.uiSettings.getDefaultIndex()
            .then(function (defaultIndex) {
              if (appName === 'discover' || appName === 'visualize' || appName === 'dashboard') {
                if (!defaultIndex) {
                  // https://github.com/elastic/kibana/issues/7496
                  // Even though most tests are using esClient to set the default index, sometimes Kibana clobbers
                  // that change.  If we got here, fix it.
                  log.debug(' >>>>>>>> WARNING Navigating to [' + appName + '] with defaultIndex=' + defaultIndex);
                  log.debug(' >>>>>>>> Setting defaultIndex to "logstash-*""');
                  return kibanaServer.uiSettings.update({
                    'dateFormat:tz': 'UTC',
                    'defaultIndex': 'logstash-*',
                    'telemetry:optIn': false
                  });
                }
              }
            })
```

Reply:
You can see that WARNING logged when you run it locally.  It's a bit harder to find it when run in CI since it's debug logging.

Reply:
ahhhh, i understand now.  I saw that error in my tests but I'm using a different archive so logstash index is an id, not the name.  Good idea on the telemetry!  Still think it need the animations turned off too.